test/Microsoft.ML.TorchSharp.Tests/QATests.cs

```diff
@@ -88,35 +88,37 @@ public void TestQALargeFileGpu()
             var model = estimator.Fit(dataView);
             var transformedData = model.Transform(dataView);
 
-            var cursor = transformedData.GetRowCursor(transformedData.Schema["Answer"], transformedData.Schema["Score"], transformedData.Schema["TrainingAnswer"], transformedData.Schema["Context"], transformedData.Schema["Question"]);
-            var answerGetter = cursor.GetGetter<VBuffer<ReadOnlyMemory<char>>>(transformedData.Schema["Answer"]);
-            var contextGetter = cursor.GetGetter<ReadOnlyMemory<char>>(transformedData.Schema["Context"]);
-            var questionGetter = cursor.GetGetter<ReadOnlyMemory<char>>(transformedData.Schema["Question"]);
-            var trainingAnswerGetter = cursor.GetGetter<ReadOnlyMemory<char>>(transformedData.Schema["TrainingAnswer"]);
-            var scoreGetter = cursor.GetGetter<VBuffer<float>>(transformedData.Schema["Score"]);
-
-            VBuffer<ReadOnlyMemory<char>> answer = default;
-            ReadOnlyMemory<char> trainingAnswer = default;
-            ReadOnlyMemory<char> context = default;
-            ReadOnlyMemory<char> question = default;
-            VBuffer<float> score = default;
-            int correct = 0;
-            int incorrect = 0;
-
-            while (cursor.MoveNext())
+            using (var cursor = transformedData.GetRowCursor(transformedData.Schema["Answer"], transformedData.Schema["Score"], transformedData.Schema["TrainingAnswer"], transformedData.Schema["Context"], transformedData.Schema["Question"]))
```

Comment:
>using (var cur [](http://example.com/codeflow?start=12&length=14)

nit:
you can do `using var cursor ....` without `( )` and not using `{` block too.